trader.js

```diff
@@ -8,24 +8,15 @@ const axios = require('axios')
 const Binance = require('node-binance-api')
 const nodemailer = require('nodemailer')
 //const TeleBot = require('telebot')
-
-//////////////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////////////
-//         PLEASE EDIT WITH YOUR BITCOINvsALTCOINS.com KEY HERE BELLOW
-//////////////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////////////
-
-const bva_key = "replace_with_your_BvA_key" 
-
+const env = require('./env')
 
 //////////////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////////
 //         PLEASE EDIT PREFERENCES BELOW
```

Comment:
There is nothing to edit anymore, is it?

Reply:
d'accord